### PR TITLE
Updated Logo

### PR DESCRIPTION
**Pull request by pixelchef**
An updated logo with Realm's new font and color palette.

- Updated source sketch file
- Updated .jpg output for README

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 
+## 0.7.3
+
+##### Breaking
+
+* None.
+
+##### Enhancements
+
+* Updated Logo
```
Comment (segiddins) [inline CHANGELOG.md]: since this doesn't change any functionality, I don't think a CHANGELOG entry is necessary?

**Comment by orta** [comment]
IMO this is good to go - new style looks nice

**Comment by istx25** [comment]
Looks good to me as well. Will merge.

**Comment by istx25** [comment]
Whoops, didn't mean to delete your branch. ❤️

**Comment by jpsim** [comment]
Nice, thanks all!

**Pull request merged**